### ISSUE TITLE
[Question] no-unnecessary-class

### ISSUE THREAD

**Issue by simPod** [palantir/tslint#4049]
```ts
export class SomeClass {
    constructor(var: string);
}
```

Reports 

> no-unnecessary-class: Every member of this class is a constructor. Use functions instead

I checked the docs but have not found any example how the correct version looks like. Could you help me please? Thanks!
